include/core/parameters.h

```diff
@@ -258,6 +258,26 @@ namespace Parameters
     parse_parameters(ParameterHandler &prm);
   };
 
+  /**
+   * @brief InterfaceSharpening - Defines the parameters for
+   * interface sharpening in the VOF solver.
+   */
+
+  struct InterfaceSharpening
+  {
+    double sharpening_threshold;
+    double interface_sharpness;
+    int    sharpening_frequency;
+    // Type of verbosity for the interface sharpening calculation
+    Verbosity verbosity;
+
+    double l2_smoothing_factor;
```

Comment:
Can you add a comment that describe the expected values and the role of these parameters?
 double sharpening_threshold;
    double interface_sharpness;
    int    sharpening_frequency;
    double l2_smoothing_factor;

Reply:
Added for each parameter. l2_smoothing_factor is removed since it was unnecessary.